net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,199 @@
+id: net.pdl2ork.PdL2Ork
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
+command: pd-wrapper.sh
+finish-args:
+# Display
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+# Audio
+  - --socket=pulseaudio
+  - --system-talk-name=org.freedesktop.RealtimeKit1
```

Comment:
```suggestion
```

Reply:
What would this do? I saw this requirement on flatpaks of other forks of this project, but am not entirely sure why this is needed, other than the fact that audio is central to this application.

Reply:
Looks like this may give rt permission to the application, which, if so, is desirable to ensure low-latency operation audio-wise.

Reply:
It will NOT work. Remove it. Those that still have it, it's a mistake. Two wrong don't make a right.

Reply:
Got it. Will remove it. Will this work perhaps at some point in the future, so that I leave it just commented out, as a reminder to revisit at a later date?